src/sentry/services/hybrid_cloud/rpc.py

```diff
@@ -447,6 +448,12 @@ def dispatch_remote_call(
     return remote_silo_call.dispatch(use_test_client)
 
 
+RPC_CACHE_ALLOW_LIST = [
+    "/api/0/internal/rpc/user_option/get_many/",
+    "/api/0/internal/rpc/subscription/get_for_organization/",
```

Comment:
I think having a local cache for the subscription makes sense as we use it in feature flag checks and there can be a bunch of those in a given request.

We also have another RPC cache in use for `get_user`. Perhaps we could use that same caching strategy for these results? The other solution is in `sentry.hybridcloud.rpc.services.caching`.